Analysis/source/ProcessAnaTuple.cxx

```diff
@@ -179,7 +179,8 @@ class ProcessAnaTuple : public EventAnalyzerCore {
                         log << "Warning: SS Anti Iso integral is too small " << hist.first << std::endl;
                         continue;
                     }
-                    const auto k_factor = osAntiIso_integral / ssAntiIso_integral;
```

Comment:
continue in the two if above should be done only in case if `qcd_ss_os_sf` is not specified